src/go/rpk/pkg/cli/cluster/partitions/list.go

```diff
@@ -83,6 +84,10 @@ List only the disabled partitions.
 
 List all in json format.
   rpk cluster partition list -a --format json
+
+Show how the partitions are distributed across brokers.
+  rpk cluster partitions list -a --broker-distribution
+  rpk cluster partitions foo bar --broker-distribution
```

Comment:
I _think_ it's ok if we enable it by default. Showing the distribution does not require an additional request. 

What do you think?

Reply:
Sorry for the delay! Yea indeed it's a good idea.